test/helpers/aws.test.js

```diff
@@ -179,17 +179,28 @@ describe('aws helper test', () => {
     });
 
     it('upload command as stream', () => {
+        const awsClientCommand = new AwsClient({
```

Comment:
In beforeEach hook, awsClient is defined with almost same parameter without segment.
https://github.com/sonic-screwdriver-cd/store/blob/4f3e9e11673a2fac38dd1a3f3728573ba371ea06/test/helpers/aws.test.js#L68-L77
Wouldn't the overwrite segment property be enough to test this helper method [like before](https://github.com/screwdriver-cd/store/blob/21aa39635a53d5aa0a5e2d724a656f6182f9b678/test/helpers/aws.test.js#L182)?

Reply:
nice! thank you